src/gql/queries/get-version.graphql

```diff
@@ -35,7 +35,7 @@ query Version($id: String!) {
         baseVersionID
         githash
         id
-        projectID
```

Comment:
why do we need to change this field? will DownstreamTasks read this field?

Reply:
turns out this one is the only one we need. I removed the others.